### ISSUE TITLE
v1.11.0 shows error: Unknown directive "aws_subscribe"

### ISSUE THREAD

**Issue by arabold** [sid88in/serverless-appsync-plugin#409]
Since upgrading to v1.11.0 the plugin reports an error when using `@aws_subscribe` directive in my `.graphql`:

```graphql
type Mutation {
  updateStatus: PipelineStatus!
}

type Subscription {
  pipelineStatusChanged: PipelineStatus! @aws_subscribe(mutations: ["updateStatus"])
}
```

It seems to be a simple validation error, maybe because the `@aws_subscribe` directive isn't declared in the linter? For now I downgraded to v1.10 again.

**Comment by bboure** [comment]
Thank you @arabold  I just noticed this today as well. Was introduced by #406

In my case, it did not prevent deploymnet.

cc @AleksaC

**Comment by AleksaC** [comment]
Overlooked that, I'll fix it tomorrow

**Comment by razor-x** [comment]
I'm not sure this is fixed, just tried it on latest version and I am getting

```
AppSync Plugin: Error: Unknown directive "aws_subscribe".

Unknown directive "aws_subscribe".
```

<img width="693" alt="image" src="https://user-images.githubusercontent.com/721372/116948707-0b6eff80-ac35-11eb-8d9f-0d873b1d444a.png">

I am using it like this

```graphql
type Subscription {
  somthing(id: ID!): SomeType
    @aws_subscribe(mutations: ["someOtherThing"])
}
```

**Comment by AleksaC** [comment]
You are right, I added `aws_subscription` instead of `aws_subscribe` 🤦‍♂️. Since appsync apparently does not validate directives during deployment the test deployment I threw together passed even though I used the wrong directive....

**Comment by bboure** [comment]
Oh noooo!! That's on me too for not double-checking and testing. 

Sorry

**Comment by bboure** [comment]
Fixed in v1.11.3 sorry
